demos/storybook/src/utils.tsx

```diff
@@ -1,31 +1,92 @@
+};
 
 export const updateTitle = (): void => {
     setTimeout(() => {
         window.top.document.title = 'PX Blue | React Components';
     }, 10);
-    (function() {
-        var link = window.top.document.querySelector("link[rel*='icon']") || document.createElement('link');
-        // @ts-ignore
-        link.type = 'image/x-icon';
-        // @ts-ignore
-        link.rel = 'shortcut icon';
-        // @ts-ignore
-        link.href = './pxblue.png';
-        window.top.document.getElementsByTagName('head')[0].appendChild(link);
-    })();
-};
-
-export const storyWrapper = (storyFn: any) => {
-    const banner = window.top.document.getElementsByClassName('simplebar-content')[1];
-    banner.setAttribute('style', 'display: unset');
-
-    // If we are currently on the 'Notes' tab.
-    if (window.top.location.href.includes('/info/')) {
-        window.top.history.replaceState(null, '', window.top.location.href.replace('/info/', '/story/'));
-        //@ts-ignore
-        banner.children[0].children[0].children[0].children[0].click(); // Click the 'Canvas' button
-    }
+
+    const link: any = window.top.document.querySelector("link[rel*='icon']") || document.createElement('link');
+    link.type = 'image/x-icon';
+    link.rel = 'shortcut icon';
+    link.href = './pxblue.png';
+    window.top.document.getElementsByTagName('head')[0].appendChild(link);
+};
+
+let prevUrl = '';
+// Auto-navigates the user to the Canvas tab when switching stories.
+export const storyWrapper = (storyFn: any): any => {
+    const currentUrl = window.top.location.href;
+    showTopBanner();
     updateTitle();
+
+    const currStoryUrl = currentUrl.replace(STORY_PATH, '').replace(NOTES_PATH, '');
+    const prevStoryUrl = prevUrl.replace(STORY_PATH, '').replace(NOTES_PATH, '');
+
+    // If user changed stories while on the Notes tab, but not from the README story.
+    if (currStoryUrl !== prevStoryUrl && !currentUrl.includes('get-read-me') && currentUrl.includes(NOTES_PATH)) {
+        selectCanvasTab();
+    }
+    prevUrl = currentUrl;
     return <>{storyFn()}</>;
 };
+
+export const getReadMeStory = (): StoryFnReactReturnType => {
+    hideTopBanner();
+    // If we are currently on the Canvas tab, go to Notes tab.  The README story never shows the Canvas.
+    if (window.top.location.href.includes(STORY_PATH)) {
+        selectNotesTab();
+    }
+    return <></>;
+};
+getReadMeStory.story = { name: README_STORY_NAME };
+
+export const storyParams = {
+    options: {
+        showPanel: true,
+    },
+    notes: {},
+};
+
+export const getReadMe = (name: string): any => {
+    const md = require(`./../../../docs/${name}`);
+
+    // Locate all relative links that use href syntax and replace them with absolute URLs.
+    md.default = (md.default).replace(/\(.\/.*md\)/g, (substring: string) => {
+        const root = 'https://pxblue-components.github.io/react-dev/';
```

Comment:
Can we update this so it uses the 'root' of the current URL rather than hardcoding react-dev?
i.e.,
- on pxblue-components.github.io/react...it should keep us on the master deployment and not direct us to dev when we click a link
- on localhost, we should stay on localhost when clicking a link, instead of going to react-dev

Reply:
Updated this, works localy; seems like it'll work in dev